tests/manage/z_cluster/nodes/test_automated_recovery_from_failed_nodes_reactive_IPI.py

```diff
@@ -176,3 +197,117 @@ def test_automated_recovery_from_failed_nodes_IPI_reactive(
 
         # Perform cluster and Ceph health checks
         self.sanity_helpers.health_check()
+
+
+@ignore_leftovers
+@tier4
+@tier4b
+@aws_platform_required
+@ipi_deployment_required
+class TestAutomatedRecoveryFromStoppedNodes(ManageTest):
+
+    osd_worker_node = None
+    extra_node = False
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request, nodes):
+
+        def finalizer():
+            if self.extra_node:
+                nodes.terminate_nodes(self.osd_worker_node, wait=True)
+                log.info(
+                    f"Successfully terminated node : "
+                    f"{self.osd_worker_node[0].name} instance"
+                )
+            else:
```

Comment:
Node is already started in the test case. Do we still need the else part?

Reply:
in case the test will fail before we are starting the node - safety.
In case the node did start in the test, this call will do nothing

Reply:
Got it. Thanks.